### PR TITLE
Fix problems in project.clj with jackson xml lib references.

### PR DESCRIPTION
**Pull request by toneillbroad**
Added RXNorm drug data to jena db and graphql interface.
Added suggesters for gene, disease and drugs.

### CODE REVIEW

**Comment by toneillbroad** [comment]
This encompasses the addition of RXNorm data to the database, and the addition of suggesters.

File: resources/property-names.edn [hunk 1]
```diff
@@ -219,7 +219,7 @@
  [:dc/accrual-periodicity
   "http://purl.org/dc/terms/accrualPeriodicity"]
  [:dc/accrual-policy "http://purl.org/dc/terms/accrualPolicy"]
- [:dc/alternative-title "http://purl.org/dc/terms/alternative"]
+ [:dc/altLernative-title "http://purl.org/dc/terms/alternative"]
```
Comment (tnavatar) [inline resources/property-names.edn:222]: This looks like a typo that snuck in?

File: src/genegraph/sink/base.clj [hunk 2]
```diff
@@ -46,10 +47,16 @@
...
+  (log/info :fn :build-suggesters! :msg :building)
+  (build-all-suggestions)
+  (log/info :fn :build-suggesters! :msg :complete))
+
 (defn initialize-db! []
   (let [res (read-base-resources)]
     (retrieve-base-data! res)
     (import-documents! res)
+    (build-suggesters!)
     (log/info :fn :initialize-db! :msg :initialization-complete)))
```
Comment (tnavatar) [inline src/genegraph/sink/base.clj:60]: Do the suggesters depend on having the curations available? This wouldn't be true at this stage...

File: resources/property-names.edn (hunk 1 above)
Comment (toneillbroad) [reply resources/property-names.edn:222]: Yup, that one snuck by me.

File: src/genegraph/sink/base.clj (hunk 2 above)
Comment (toneillbroad) [reply src/genegraph/sink/base.clj:60]: That explains why I'm not seeing curations when I create-migration. Thanks for pointing this out.

**Pull request merged**